modules/standard/nano/nano_controller.py

```diff
@@ -34,7 +34,9 @@ def nano_cmd(self, request, search, named_params):
 
         blob = ""
 
-        if count == 1:
+        if count == 0:
+            return "No results found for <highlight>%s</highlight> " % search
```

Comment:
Remove the space at the end, and add a period

Reply:
Updated, and matched the wording from !items output.